wagtail/wagtailadmin/templates/wagtailadmin/admin_base.html

```diff
@@ -25,6 +25,10 @@
     <script src="{% static 'wagtailadmin/js/vendor/jquery.dlmenu.js' %}"></script>
     <script src="{% static 'wagtailadmin/js/vendor/tag-it.js' %}"></script>
     <script src="{% static 'wagtailadmin/js/core.js' %}"></script>
+    <script>
```

Comment:
Consider moving this a line up, so it's paired with the `tag-it.js` library inclusion.